test/switch_test.js

```diff
@@ -161,9 +210,8 @@ describe.skip('switch', () => {
           case c:
             return d;
         }
-      };
-
-      e;
+      }
+      ;e;
```

Comment:
Need some help figuring out this one.

These are the patcher edits:
```
FunctionPatcher INSERT 4 "function" BEFORE "->"
FunctionPatcher INSERT 4 "() " BEFORE "->"
FunctionPatcher OVERWRITE [4, 6) "->" → "{"
IdentifierPatcher INSERT 16 "(" BEFORE "b\n"
IdentifierPatcher INSERT 17 ")" BEFORE "\n "
IdentifierPatcher INSERT 17 " {" BEFORE "\n "
SwitchCasePatcher REMOVE [22, 27) "when "
IdentifierPatcher INSERT 27 "case " BEFORE "c\n"
IdentifierPatcher INSERT 28 ":" BEFORE "\n "
BlockPatcher INSERT 35 "return " BEFORE "d\n"
BlockPatcher INSERT 36 ";" BEFORE "\ne"
SwitchPatcher INSERT 36 "\n  }" BEFORE "\ne"
BlockPatcher INSERT 37 ";" BEFORE "e"  <---
BlockPatcher INSERT 36 "\n}" BEFORE "\ne" <---
BlockPatcher INSERT 37 ";" BEFORE "e"  <---
BlockPatcher INSERT 38 ";" BEFORE ""
```

It looks like the BlockPatcher semicolon insertion is conflicting with the right brace insertion.